core/nodejs6Action/Dockerfile

```diff
@@ -36,6 +36,7 @@ glob@7.1.1 \
 gm@1.23.0 \
 lodash@4.17.2 \
 log4js@0.6.38 \
+ibm_db@2.0.0 \
```

Comment:
This will not installed.
In addition, it adds large amount of files that are not need it see #2311